http-client.ts

```diff
@@ -176,7 +183,7 @@ export class HttpClient implements Server.IHttpClient {
 		return response;
 	}
 
-	private async setResponseResult(result: IPromiseActions<Server.IResponse>, timerId: number, resultData: { response?: Server.IRequestResponseData, body?: string, err?: Error,  }): Promise<void> {
+	private async setResponseResult(result: IPromiseActions<Server.IResponse>, timerId: number, resultData: { response?: Server.IRequestResponseData, body?: string, err?: Error, }): Promise<void> {
```

Comment:
I think we don't need the `,` after `err?: Error` in the `resultData` type definition.